packages/shared-fixtures/fixtures/typescript/basics/type-import-type.src.ts

```diff
@@ -1,2 +0,0 @@
-type A = typeof import('A');
-type B = import('B').X<Y>;
```

Comment:
shared-fixtures has been removed in v6 and test is already correctly migrated, this is most leftover from merge conflict